lib/backend/firestore/firestorebk_test.go

```diff
@@ -52,15 +53,25 @@ func TestMarshal(t *testing.T) {
 	assert.NoError(t, err)
 }
 
-var commonFirestoreParams = map[string]interface{}{
-	"collection_name":                   "tp-cluster-data-test",
-	"project_id":                        "tp-testproj",
-	"endpoint":                          "localhost:8618",
-	"purgeExpiredDocumentsPollInterval": time.Second,
+// var commonFirestoreParams = map[string]interface{}{
+// 	"collection_name":                   "tp-cluster-data-test",
+// 	"project_id":                        "tp-testproj",
+// 	"endpoint":                          "localhost:8618",
+// 	"purgeExpiredDocumentsPollInterval": time.Second,
+// }
+
+func firestoreParams(t *testing.T) map[string]interface{} {
```

Comment:
Maybe:

```suggestion
type firestoreConfig map[string]interface

func newFirestoreConfig(t *testing.T) firestoreConfig {
```

and use `firestoreConfig` throughout?